test/basic/init_funcs_example.c

```diff
@@ -0,0 +1,38 @@
+#include "smack.h"
+
```

Comment:
Please create a failing version of this and call it init_funcs_example_fail.c.
An assertion for that failing example could be:
__SMACK_code("assert($processStatus[@] != $process_uninitialized);", idx);